lib/Dialect/ESI/ESIPasses.cpp

```diff
@@ -119,6 +120,29 @@ ESIHWBuilder::ESIHWBuilder(Operation *top)
     setInsertionPoint(&region.front(), region.front().begin());
 }
 
+static StringAttr constructUniqueSymbol(Operation *tableOp,
```

Comment:
@lattner Is there an existing function to "uniquify" symbol names? When I don't do this, I get symbol redefinition errors.